src/api/transaction/suspay-finish.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+'use strict';
+const _ = require('lodash');
+const utils = require('./utils');
+const validate = utils.common.validate;
+const Transaction = utils.common.core.Transaction;
+
+function createSusPayFinishTransaction(account, payment) {
+  validate.address(account);
+  validate.suspendedPaymentFinish(payment);
+
+  const transaction = new Transaction();
+  transaction.susPayFinish({
+    from: account,
+    owner: payment.owner,
+    offerSequence: payment.offerSequence
+  });
+
+  if (payment.method) {
+    transaction.setMethod(payment.method);
+  }
+  if (payment.digest) {
+    transaction.setDigest(payment.digest);
+  }
+  if (payment.proof) {
+    transaction.setProof(payment.proof);
+  }
+
+  if (payment.memos) {
+    _.forEach(payment.memos, memo =>
+      transaction.addMemo(memo.type, memo.format, memo.data)
+    );
+  }
+  return transaction;
+}
+
+function prepareSusPayFinishAsync(account, payment, instructions, callback) {
+  const transaction = createSusPayFinishTransaction(account, payment);
+  utils.createTxJSON(transaction, this.remote, instructions, callback);
+}
+
+function prepareSusPayFinish(account: string, payment: Object,
+instructions = {}) {
+  return utils.promisify(prepareSusPayFinishAsync.bind(this))(
```

Comment:
Oh wait, can we just do `utils.promisify(fn).apply(this, arguments)` in most places?